app/controllers/signatures/ongoing.js

```diff
@@ -20,4 +22,41 @@ export default class SignaturesOngoingController extends Controller {
   @tracked page = 0;
   @tracked size = PAGINATION_SIZES[1];
   @tracked sort = 'decision-activity.start-date';
+
+  isConfidential = (accessLevel) => {
+    return [
+      CONSTANTS.ACCESS_LEVELS.INTERN_SECRETARIE,
+      CONSTANTS.ACCESS_LEVELS.VERTROUWELIJK,
+    ].includes(accessLevel.get('uri'));
+  }
+
+  getMandateeNames = async (signFlow) => {
+    const signSubcase = await signFlow.signSubcase;
+    const signSigningActivities = await signSubcase.signSigningActivities;
+    const mandatees = await Promise.all(
+      signSigningActivities.map((activity) => activity.mandatee)
+    );
+    const persons = await Promise.all(
+      mandatees
+        .sort((m1, m2) => m1.priority - m2.priority)
+        .map((mandatee) => mandatee.person)
+    );
+    return persons.map((person) => person.fullName);
```

Comment:
I think this is right? For agenda it makes sense to show them based on priority.
For signatures, is the `requestBy` more important or not? Leave as-is for now.